src/Resources/contao/library/Contao/Widget.php

```diff
@@ -1181,7 +1181,7 @@ public static function optionSelected($strOption, $varValues)
 			return '';
 		}
 
-		return (\is_array($varValues) ? \in_array($strOption, $varValues) : $strOption == $varValues) ? ' selected' : '';
+		return \in_array($strOption, array_map('strval', (array) $varValues), true) ? ' checked' : '';
```

Comment:
Why did you change `selected` to `checked` here?

Reply:
I copy your code...
https://github.com/contao/contao/issues/1022#issuecomment-575210705

Reply:
I used your code from https://github.com/contao/contao/issues/1022#issuecomment-564702570 as the basis.
But making a pull request with just a copy of my code doesn’t bring us much further. What we need to do is described here: https://github.com/contao/contao/issues/1022#issuecomment-695349026

Reply:
hmm something is on the wrong rail here - I'll check it again